.markdownlintrc

```diff
@@ -1,10 +1,13 @@
 {
   "default": true,
-  "MD003": { "style": "atx" },
+  "MD002": false,
   "MD007": { "indent": 4 },
-  "MD013": { "line_length": 200 },
+  "MD013": { "line_length": 500 },
```

Comment:
Yeah. We would need a smarter parser. I wouldn't worry about it. Keep the line length sufficiently long and it will be enough. `500` is a starting point.